spec/helpers/transaction_history_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "spec_helper"
+
+describe TransactionHistoryHelper do
+  describe "#chosen_field"
```

Comment:
Intentionally left blank. This whole file is tangential: when I started adding Disputed Orders to the Billing section I ran across the transaction search helper, and noticed there were no specs. So I wrote some, and left some pending.